vignettes/profiles-with-rank.Rmd

```diff
@@ -0,0 +1,145 @@
+
+Additionally, these functions can reveal what combination of attributes describes a group that shows a specific behaviour (i.e for example, what organisational attributes describe employees suffering from excessive workload, receiving insufficient coaching, etc). As business leaders often want to develop targeted interventions to address specific needs, these functions can be a great way to inform action planning and transformation initiatives.
+
+This article is divided in four sections. After this introduction, section two introduces the rank functions generally an how to apply them to person query output. In the third section, we explore their use  for profiling - i.e. to identify in which groups are specific behaviour is most common. The final section, includes a list of additional use cases that can benefit from this method. 
+
+
+
+## Rank Functions
+
+### Baseline analysis
+
+To get started, let's open a new session in R and load the [**wpa** R package](https://microsoft.github.io/wpa). We will also use **dplyr** to facilitate data manipulation:
+
+```{r message=FALSE, warning=FALSE}
+library(wpa)
+library(dplyr)
+```
+
+Using the `sq_data` demo dataset (it contains a weekly person query output) let's use the `collaboration_rank()` function:
+
+```{r message=FALSE, warning=FALSE}
+
+sq_data %>% 
+collaboration_rank(return="table")
+
+```
+
+The output table contains the results from the automatic group exploration. The rank function has taken all available organisational attributes in this dataset ("FunctionType", "Organization", "LevelDesignation", and "IsActive") and calculated the average collaboration hours for each group within each one of them. The table shows the results for all groups analysed, ranked them from highest to lowest. 
+
+To see the entire table, we can add the function `print` to the command above. 
+
+```{r message=FALSE, warning=FALSE}
+sq_data %>% 
+collaboration_rank(return="table") %>%  
+print(n=30)
+```
+
+This analysis shows us that the 82 employees labeled as **"FunctionType = Sales"** have the largest collaboration hours, with a weekly average of 31 hours. In second place are those in **"Organization = Inventory Management"** with 26.2 hours. On the lower end, the employees labelled as **"Organization = Biz Dev"** appear to have the lowest collaboration time. 
+
+This output is useful to identify groups that are worth investigating (e.g. "Sales"). It also reveals that splits by "FunctionType" and "Organization" are more interesting than those by "LevelDesignation", as they have more variability (All groups in "LevelDesignation" have  relatively similar levels of collaboration, between 18 and 22 hours a week). 
+
+Rank functions can also display this information in visual form. We can get that by changing the `return` argument from "table" to "plot":
+
+```{r message=FALSE, warning=FALSE}
+sq_data %>% 
+collaboration_rank(return="plot")
```

Comment:
Again adding indentation and spacing: 

```suggestion
```{r message=FALSE, warning=FALSE}
sq_data %>% 
    collaboration_rank(return = "plot")
```